src/org/labkey/test/util/TestDataUtils.java

```diff
@@ -191,4 +191,12 @@ protected boolean shouldQuote(String value)
             return StringUtils.containsAny(value, _escapedChars);
         }
     }
+
+    public static final String[] DECODED = {"\\", "$", "/", "&", "}", "~", ",", "."};
+    public static final String[] ENCODED = {"\\\\", "\\$", "\\/", "\\&", "\\}", "\\~", "\\,", "\\."};
+    public static String getEscapedNameExpression(String encoded)
+    {
+        return StringUtils.replaceEach(encoded, DECODED, ENCODED);
```

Comment:
I think you can just reference `QueryKey.DECODED` and `QueryKey.ENCODED` here since they are public and static.